pkg/reader/config_poller_test.go

```diff
@@ -121,23 +127,14 @@ func setupInitialData(ctx context.Context, cache *configPoller, reader *reader_m
 
 	// Setup source chain configs
 	sourceChains := []cciptypes.ChainSelector{chainB, chainC}
-
-	result1 := &types.BatchReadResult{ReadName: consts.MethodNameGetSourceChainConfig}
-	result1.SetResult(&cciptypes.SourceChainConfig{IsEnabled: true, OnRamp: cciptypes.UnknownAddress{1, 2, 3}}, nil)
-	result2 := &types.BatchReadResult{ReadName: consts.MethodNameGetSourceChainConfig}
-	result2.SetResult(&cciptypes.SourceChainConfig{IsEnabled: true, OnRamp: cciptypes.UnknownAddress{4, 5, 6}}, nil)
-
-	response := types.BatchGetLatestValuesResult{
-		types.BoundContract{Name: consts.ContractNameOffRamp}: {
-			*result1, *result2,
-		},
-	}
-
-	reader.On("ExtendedBatchGetLatestValues",
-		mock.Anything,
-		mock.Anything,
-		false,
-	).Return(response, []string{}, nil).Once()
+	sourceConfig1 := &cciptypes.SourceChainConfig{IsEnabled: true, OnRamp: cciptypes.UnknownAddress{1, 2, 3}}
+	sourceConfig2 := &cciptypes.SourceChainConfig{IsEnabled: true, OnRamp: cciptypes.UnknownAddress{4, 5, 6}}
+	destChainAccessorMock := cache.chainAccessors[chainA].(*ccipocr3.MockChainAccessor)
+	destChainAccessorMock.On("GetSourceChainConfigs", mock.Anything, mock.Anything).Return(
```

Comment:
nit: better type safety with this version
```suggestion
	destChainAccessorMock.EXPECT().GetSourceChainConfigs(mock.Anything, mock.Anything).Return(
```